### PR TITLE
type hint faker generator

### PR DESCRIPTION
**Pull request by browner12**
allow IDE to help with autocompletion

### CODE REVIEW

File: database/factories/ModelFactory.php [hunk 1]
```diff
@@ -11,7 +11,7 @@
 |
 */
 
-$factory->define(App\User::class, function ($faker) {
+$factory->define(App\User::class, function (\Faker\Generator $faker) {
```
Comment (GrahamCampbell) [inline database/factories/ModelFactory.php]: No leading slash please.

**Comment by browner12** [comment]
sure thing.

is that a PSR styling thing? or some other reason?

**Comment by GrahamCampbell** [comment]
The leading slash is actually wrong - that's why it needs removing. PHP just supports it anyway because people commonly use it by mistake.

**Comment by GrahamCampbell** [comment]
It's only use is to tell php to look to the global namespace while inside a namespace. All fully qualified class names are defined without a leading slash. :)

**Comment by browner12** [comment]
good to know!

should be updated

**Comment by GrahamCampbell** [comment]
:)

**Comment by GrahamCampbell** [comment]
:+1: from me

**Pull request merged**